Scoop/Controller/Ride Posting/PostRideSummaryViewController.swift

```diff
@@ -55,20 +55,25 @@ class PostRideSummaryViewController: UIViewController {
         fatalError("init(coder:) has not been implemented")
     }
     
+    //TODO: Not good practice, but temporary fix. Will Debug later
+    override func viewDidAppear(_ animated: Bool) {
+        updateBackButton()
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .white
         guard let creator = currentRide?.creator else { return }
-        navigationItem.title = "\(creator.firstName)'s Ride"
         setUpStackView()
         setUpButton()
         setUpLabelFont()
         setUpTextFont()
         setUpIcons()
+        updateBackButton()
     }
     
     private func setUpStackView() {
-        let stackViewMultiplier = 0.05
+        let stackViewMultiplier = 0.0
```

Comment:
Am i tripping or is that 0.0? Is there a reason why this multiplier is 0?

Reply:
I think I have that there in case they changed the spacing of the app later on. The distance between the top of the screen and the stack view has changed multiple times lol